bin/cron/upload_to_firebase

```diff
@@ -44,10 +46,31 @@ def get_weather_data
   return records
 end
 
+def parse_csv(filename)
+  records = []
+  id = 1
```

Comment:
What happens if we run this again? Will the records be overwritten? Wondering because the id numbers will be reused.

Reply:
Yes, records will be overwritten.

Reply:
👍